packages/twenty-server/src/engine/core-modules/user-workspace/user-workspace.resolver.ts

```diff
@@ -11,16 +11,17 @@ import { User } from 'src/engine/core-modules/user/user.entity';
 import { Workspace } from 'src/engine/core-modules/workspace/workspace.entity';
 import { AuthUser } from 'src/engine/decorators/auth/auth-user.decorator';
 import { WorkspaceAuthGuard } from 'src/engine/guards/workspace-auth.guard';
+import { WorkspaceInvitationService } from 'src/engine/core-modules/workspace-invitation/services/workspace-invitation.service';
+import { WorkspaceInviteTokenInput } from 'src/engine/core-modules/auth/dto/workspace-invite-token.input';
 
 @UseGuards(WorkspaceAuthGuard)
 @Resolver(() => UserWorkspace)
 export class UserWorkspaceResolver {
   constructor(
     @InjectRepository(Workspace, 'core')
     private readonly workspaceRepository: Repository<Workspace>,
-    @InjectRepository(User, 'core')
-    private readonly userRepository: Repository<User>,
     private readonly userWorkspaceService: UserWorkspaceService,
+    private readonly workspaceInvitationService: WorkspaceInvitationService,
   ) {}
 
   @Mutation(() => User)
```

Comment:
we will likely need to rename this mutation to be consistent
addUserToWorkspaceByInviteHash